services/ipcExec.js

```diff
@@ -3,27 +3,28 @@ const Promise = require('bluebird'),
   _ = require('lodash'),
   ipc = require('node-ipc');
 
-module.exports = async (method, params) => {
-
-  const ipcInstance = new ipc.IPC;
-
-  Object.assign(ipcInstance.config, {
-    id: `${Date.now()}${_.random(Math.pow(2, 32))}`,
-    socketRoot: config.node.ipcPath,
-    retry: 1500,
-    sync: true,
-    silent: true,
-    unlink: true,
-    maxRetries: 3
-  });
-
-  await new Promise((res, rej) => {
+const ipcInstance = new ipc.IPC;
```

Comment:
have you read the code?, i've moved it to the method's body, in order to create a fresh connection on each request!